crates/sui-core/src/authority.rs

```diff
@@ -1331,9 +1331,15 @@ impl AuthorityState {
                 epoch_store,
             )
             .tap_err(|e| info!("process_certificate failed: {e}"))
-            .tap_ok(
-            |(fx, _)| debug!(?tx_digest, fx_digest=?fx.digest(), "process_certificate succeeded"),
-        )?;
+            .tap_ok(|(fx, _)| {
+                let elapsed = execution_start_time.elapsed().as_micros() as f64;
```

Comment:
should this be inline the debug! macro so that it's not computed unless debug logging is on here?

`.as_secs_f64() * 1000` would remove the need for a cast, not sure how much we care